addons/calendar/models/res_users.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+
+from odoo import api, fields, models, modules, _
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    def _systray_get_event_domain(self):
+        start_dt = datetime.datetime.utcnow()
+        start_date = datetime.date.today()
+        end_dt = datetime.datetime.combine(start_date, datetime.time.max)
+
+        return ['&', '|',
```

Comment:
not sure why, but i think this actually broke the feature. the systray_get_data call does not return meetings anymore